cmd/manta-init.js

```diff
@@ -643,7 +643,9 @@ var pipelineFuncs = [
 
 		var extra = {
 			metadata: {
-				MANTAV: 1
+				// This authoritatively indicates that this is a
```

Comment:
Offline, we talked about potentially having a guard around the code here to prevent inadvertently re-manta-initing with the wrong branch-flavour of sdc-manta, Trent opened MANTA-4809 for that. That doesn't have to be this pull request, but we should try to add it soon.